omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/ChangePasswordController1_8Test.java

```diff
@@ -72,10 +72,14 @@ public void testChangeUsersOwnPassword() throws Exception {
 		
 		String oldPassword = "SomeOtherPassword123";
 		String newPassword = "newPassword9";
-		
-		MockHttpServletResponse response = handle(newPostRequest(PASSWORD_URI, "{\"newPassword\":\"" + newPassword + "\""
-		        + "," + "\"oldPassword\":\"" + oldPassword + "\"}"));
-		assertEquals(200, response.getStatus());
+		try {
```

Comment:
Why are these proxy privileges needed in tests?  Didn't the actual endpoint have these added to it to support this?